pages/osx/xcode-select.md

```diff
@@ -0,0 +1,24 @@
+# xcode-select
+
+> Switch between different versions of Xcode and the included developer tools.
+> Also used to update the path to Xcode if it is moved after installation.
+
+- Install Xcode's command line tools:
+
+`xcode-select --install`
+
+- Select a given path as the active developer directory."
```

Comment:
Actually, I just noticed something here: 
```suggestion
- Select a given path as the active developer directory.
```

Can you please apply that change?

Reply:
Boom, done.